src/Collections/PropertyCollection.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
```

Comment:
extra whitespace?

Reply:
So when you create a class in phpstorm it does that white spacing for you and we haven't touched that. it would be in every file we have. should we still change that?